src/families/tron/AccountBalanceSummaryFooter.js

```diff
@@ -1,118 +1,127 @@
 // @flow
 
-import React from "react";
+import React, { useCallback, useMemo, useState } from "react";
 import { View, TouchableOpacity, ScrollView, StyleSheet } from "react-native";
-import { Trans } from "react-i18next";
-import { formatCurrencyUnit } from "@ledgerhq/live-common/lib/currencies";
+import { translate } from "react-i18next";
+import type { TFunction } from "react-i18next";
+import {
+  formatCurrencyUnit,
+  getCryptoCurrencyById,
+} from "@ledgerhq/live-common/lib/currencies";
+import { getCryptoCurrencyIcon } from "@ledgerhq/live-common/lib/reactNative";
+import invariant from "invariant";
 import colors from "../../colors";
 import LText from "../../components/LText";
 import Info from "../../icons/Info";
+import InfoModal from "../../modals/Info";
+import type { ModalInfo } from "../../modals/Info";
+import FreezeIcon from "../../icons/Freeze";
+import BandwidthIcon from "../../icons/Bandwidth";
+import EnergyIcon from "../../icons/Energy";
 
-type Props = {
-  account: any,
-  countervalue: any,
-};
+interface Props {
+  account: any;
+  countervalue: any;
+  t: TFunction;
+}
 
 const formatConfig = {
   disableRounding: true,
   alwaysShowSign: false,
   showCode: true,
 };
 
-const AccountBalanceSummaryFooter = ({ account }: Props) => {
-  if (!account.tronResources) return null;
+type InfoName = "available" | "frozen" | "bandwidth" | "energy";
+
+function AccountBalanceSummaryFooter({ account, t }: Props) {
+  const [infoName, setInfoName] = useState<InfoName | typeof undefined>();
```

Comment:
`InfoName | typeof undefined` => `?InfoName`

Reply:
yea it can be, but personally I rather not using null. When we transform to TS in the future, there's no maybe type so we need to define as `InfoName | undefined | null`